tck-runner/pom.xml

```diff
@@ -323,5 +320,47 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>security-manager</id>
+            <activation>
+                <property>
+                    <name>with-security-manager</name>
+                    <value>true</value>
+                </property>
+            </activation>
+            <properties>
+                <arquillian.protocol>LocalSecurityManagerTesting</arquillian.protocol>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>org.hibernate.beanvalidation.tck</groupId>
+                    <artifactId>beanvalidation-standalone-container-adapter</artifactId>
+                    <version>${tck.version}</version>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <systemProperties>
+                                <property>
+                                    <name>validation.provider</name>
+                                    <value>${validation.provider}</value>
+                                </property>
+                                <property>
+                                    <name>excludeIntegrationTests</name>
+                                    <value>true</value>
+                                </property>
+                            </systemProperties>
+                            <argLine>-Djava.security.manager -Djava.security.policy=${project.build.directory}/test-classes/test.policy -Djava.security.debug=access</argLine>
```

Comment:
> I've changed it to one profile, running with or without the SM being enabled based on the property. The default is now running with the SM, but it can be disabled l

+1 Right, that's what I had in mind as well.